primitives/src/constants.rs

```diff
@@ -85,3 +85,7 @@ pub const SD_PALLET_ID: PalletId = PalletId(*b"zge/sedp");
 pub const MAX_ASSETS: u16 = MAX_CATEGORIES + 1;
 /// Pallet identifier, mainly used for named balance reserves.
 pub const SWAPS_PALLET_ID: PalletId = PalletId(*b"zge/swap");
+
+// Treasury
+/// Pallet identifier, used to derive treasury account
+pub const TREASURY_PALLET_ID: PalletId = PalletId(*b"zge/swap");
```

Comment:
please correct the ID, instead of swap we should use TRES or somthing?

Reply:
Good catch :+1: It was `PalletId(*b"zge/tsry")` before. Fixed in https://github.com/zeitgeistpm/zeitgeist/pull/661/commits/a5ed0762ebc5695760d4d9107f2278d4ec19c685.